internal/metrics/metricsManager.go

```diff
@@ -77,6 +93,10 @@ const (
 	imageLabel                 = "imageName"
 	hosts                      = "hosts"
 	clusters                   = "clusters"
+	release                    = "release"
+	status                     = "status"
```

Comment:
Could you please tell me why you added those labels? I didn't see that you used them in the code

Reply:
You are right, we can remove these.

I was originally working on a different approach (actually something similar to what @rccrdpccl is suggesting now) but decided to make things simpler.